controllers/nnf_workflow_controller_helpers.go

```diff
@@ -745,6 +746,26 @@ func findCopyOutDirectiveIndexByName(workflow *dwsv1alpha1.Workflow, name string
 	return -1
 }
 
+// Returns the directive index matching the container directive whose storage field(s) reference
+// the provided name argument, or -1 if not found.
+func findContainerDirectiveIndexByName(workflow *dwsv1alpha1.Workflow, name string) int {
+	for idx, directive := range workflow.Spec.DWDirectives {
+		if strings.HasPrefix(directive, "#DW container") {
```

Comment:
You can use `parameters["command"] == container` instead of matching on "#DW container". That will prevent any whitespace differences from tripping things up.